lib/seed/Seeder.js

```diff
@@ -103,22 +85,14 @@ class Seeder {
     );
   }
 
-  // Generates the stub template for the current seed file, returning a compiled template.
```

Comment:
@lorefnon I assume we stopped using it after your refactoring?

Reply:
Yeah, i moved the lodash template usage to a shared utility which we can potentially use from other places as well.